safe-global/web-core
# fix: hydrate store after mount

## What it solves

Hydration render errors when loading persisted state from `localStorage`.

## How this PR fixes it

Instead of initialising Redux with a `preloadedStore`, a `storeHydrator` HOC now wraps the app, hydrating the store after mount.

## How to test it

Open the address book and ensure entries exist. Refresh the page and observe no hydration errors.
